sljit_src/sljitNativeARM_64.c

```diff
@@ -32,7 +32,7 @@
 
 SLJIT_API_FUNC_ATTRIBUTE const char* sljit_get_platform_name(void)
 {
-	return "ARM-64" ARM_ATOMIC_INFO SLJIT_CPUINFO;
+	return "ARM" ARM_ATOMIC_INFO SLJIT_CPUINFO;
```

Comment:
Why 64 is removed?

Reply:
I know, but I think this is true for other CPU-s as well.

Reply:
naming is not ARM's forte, so the "safest" is to just use the company name IMHO; the more consistent (although a little confusing, because there is a huge marketing push to use the instruction set name instead, which is called "execution state" in ARM's lingo, because the most popular consumer CPU only support that, and some newer contenders, specially in the server space are an even newer model of ARM v9) would be to use `ARM v8-A`, or maybe "ARM v8" so that the other two profiles which might be working already are included.